<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="codeLabel">
        <property name="text">
         <string>代码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="showCode"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="inputLable">
        <property name="text">
         <string>输入代码行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputData"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="currentVaribleLable">
        <property name="text">
         <string>当前变量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="currentVarible_2"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="treeLable">
          <property name="text">
           <string>语句与语法树</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="showTree"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="resultLable">
        <property name="text">
         <string>运行结果</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="showResult"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadButton_2">
        <property name="text">
         <string>Load(载入代码)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runbutton">
        <property name="text">
         <string>run(执行代码)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>clear(清空代码)</string>
        </property>
        <property name="autoRepeatDelay">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="debug">
        <property name="text">
         <string>debug/step(调试/单步)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
